1.py

```diff
@@ -0,0 +1,107 @@
+import glob
+import re
+import sys
+from time import localtime
+
+# 1 line: Output
+print('Hello, world!')
+# 2 lines: Input, assignment
+name = input('What is your name?\n')
+print('Hi, %s.' % name)
+# 3 lines: For loop, built-in enumerate function, new style formatting
+friends = ['john', 'pat', 'gary', 'michael']
+for i, name in enumerate(friends):
+    print("iteration {iteration} is {name}".format(iteration=i, name=name))
+# 4 lines: Fibonacci, tuple assignment
+parents, babies = (1, 1)
+while babies < 100:
+    print('This generation has {0} babies'.format(babies))
+    parents, babies = (babies, parents + babies)
+# 5 lines: Functions

```

Comment:
here you have 0 lines between new task docstring and 2 empty lines between task description and task solution. 
I suggest next pattern: 
```python
   pass  # last line of previous task
    
 
   # next task docstring
   pass  # my next task first line  
```